### ISSUE TITLE
Improve identification of the "preview documentation content changes" environment

### ISSUE THREAD

**Issue by tbouffard** [bonitasoft/bonitasoft.github.io#111]
This is an improvement after #50 

Except with the url (something like `https://bonitasoft-bonita-ici-doc-build_preview-pr-124.surge.sh/bici/latest/`) and the page title (but it is not very visible in browser tab), we can not see at a first glance that we are looking as a preview doc content changes environment:

![image](https://user-images.githubusercontent.com/27200110/108527623-cb091200-72d2-11eb-80b0-8f1abd7e910e.png)

It will be nice to have a visual information like a different banner color and the component + pr number we are checking.
The title in the antora playbook used by the preview is set with this information, but we don't display the antora site title.
We could use it instead of the bonitasoft logo (at the top left of the page).
If possible, we should trigger this `preview mode` by setting a custom property/attribute in the antora playbook to make this explicit and avoid side effects in the future

**Comment by tbouffard** [comment]
It seems that the Antora site.keys is intended for such usage: https://docs.antora.org/antora/2.3/playbook/site-keys/
